webui/vendor/Bareos/library/Bareos/BSock/BareosBSock.php

```diff
@@ -897,7 +897,7 @@ private function cram_md5_challenge($password)
      * @param $api
      * @return string
      */
-    public function send_command($cmd, $api=0)
+    public function send_command($cmd, $api = 0)
     {
         $result = "";
         $debug = "";
```

Comment:
Could the default case no be
```
self::send(".api " . $api);
$debug = self::receive_message();
```
so that we do not need `case 1` or is there an edge case that im missing ?

Reply:
Do we still want to keep code that supports bareos `15.2.2` ? Otherwise i would delete the `preg_match`.

Also why do we only catch exceptions when `api = 2`  ?